types/bnctypes/address.go

```diff
@@ -0,0 +1,71 @@
+package bnctypes
+
+import (
+	"crypto/ecdsa"
+	"encoding/hex"
+
+	"github.com/binance-chain/go-sdk/common/types"
+	"github.com/binance-chain/go-sdk/types/msg"
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/btcsuite/btcutil"
+)
+
+type Recipient struct {
+	Address Address
+	Coins   Coins
+}
+
+type Recipients []Recipient
+
+type Address struct {
+	Network    Network
+	PubKeyHash []byte
+}
+
+func (recipients Recipients) SendTx(from Address) msg.SendMsg {
+	types.Network = from.Network.ChainNetwork()
+	var inputCoins types.Coins
+	ops := make([]msg.Output, len(recipients))
+	for i, recipient := range recipients {
+		ops[i] = msg.Output{
+			Address: recipient.Address.AccAddress(),
+			Coins:   types.Coins(recipient.Coins),
+		}
+		ops[i].Coins = ops[i].Coins.Sort()
+		inputCoins = inputCoins.Plus(ops[i].Coins)
+	}
+	ips := []msg.Input{{Address: from.AccAddress(), Coins: inputCoins}}
+	return msg.SendMsg{Inputs: ips, Outputs: ops}
+}
+
+func AddressFromBech32(address string, network Network) (Address, error) {
+	types.Network = network.ChainNetwork()
```

Comment:
This is dangerous to change the global variable of the package. Maybe reimplement the function here along with the type you need.